### ISSUE TITLE
[BUG]: _get_real_channel_link_handle_empty_real_path: command not found with --channel-link

### ISSUE THREAD

**Issue by tedwardd** [pystardust/ytfzf#682]
### Describe the bug
ytfzf returns `/usr/bin/ytfzf: line 751: _get_real_channel_link_handle_empty_real_path: command not found` when using `@channel` url format. Looking at the ytfzf source it appears that `_get_real_channel_link()` should be able to handle this. It seems to me that the call to `_get_real_channel_link_handle_empty_real_path` on L751 might be getting called via some path which does not first define the function (though that seems like it would be strange to me)...

```bash
$ sed -n 706,710p `which ytfzf`
        http?://*/@*)
                domain=${_input_link#https://}
                domain=${domain%%/*}
                printf "https://www.youtube.com/channel/%s\n" "$(_get_request "$_input_link" | sed -n 's/.*itemprop="channelId" content="\([^"]*\)".*/\1/p')"
                ;;
```
### To Reproduce
```
[elw@blackbook2 ~]$ ytfzf --channel-link=https://www.youtube.com/@tested
https://www.youtube.com/channel/
/usr/bin/ytfzf: line 751: _get_real_channel_link_handle_empty_real_path: command not found
```

### Expected behavior
```
[elw@blackbook2 ~]$ ytfzf --channel-link=https://www.youtube.com/@tested
https://www.youtube.com/channel/UCiDJtJKMICpb9B1qf7qjEOA
```

### Information
 - OS:
 ```
 $ uname -a
Linux blackbook2 6.3.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 14 Jun 2023 20:10:31 +0000 x86_64 GNU/Linux
```
 - Terminal: 
 ```
 $ bash --version
GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
```
 - Ytfzf version:
 ```
 $ ytfzf --version
ytfzf: 2.6.0
```
 - Output of `readlink $(which sh)`:
 ```
 $ readlink $(which sh)
bash
```

### Additional context
In the above example, using the Tested channel on Youtube, using `@tested` produced the error while changing it to `/c/tested` produced the channel url. Unfortunately, this does not work for every channel so is only a work around for some.

**Comment by Euro20179** [comment]
Yes, this should have worked with your link, I have pushed a fix to the development branch.

**Comment by tedwardd** [comment]
@Euro20179 I've pulled down the devleopment branch with your change, now it returns and empty line:
```
[elw@blackbook2 ytfzf]$ ytfzf --version
ytfzf: git-2014.206a983 
[elw@blackbook2 ytfzf]$ ytfzf --channel-link https://www.youtube.com/@UsagiElectric
```

**Comment by Euro20179** [comment]
I think you need to do `ytfzf --channel-link=https://www.youtube.com/@UsagiElectric`

**Comment by mojibake-umd** [comment]
@Euro20179  Looks like you released fix in v2.6.1 so this can be closed.
